scarb/src/core/manifest/toml_manifest.rs

```diff
@@ -456,6 +457,8 @@ impl TomlManifest {
 
         let targets = self.collect_targets(package.name.to_smol_str(), root)?;
 
+        let publish = package.publish.unwrap_or(true);
```

Comment:
was this discussed somewhere? I'm trying to understand how desirable would that be for users, maybe publishable packages should be publishable only by explicitly stating so?

Reply:
I considered this as well, but it was described this way in the issue and cargo also has this as true by default